desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -20,6 +24,35 @@ TabDiveInformation::TabDiveInformation(QWidget *parent) : TabBase(parent), ui(ne
 	QStringList atmPressTypes { "mbar", get_depth_unit() ,"use dc"};
 	ui->atmPressType->insertItems(0, atmPressTypes);
 	pressTypeIndex = 0;
+	// This needs to be the same order as enum dive_comp_type in dive.h!
+	QStringList types = QStringList();
```

Comment:
The initialization is pointless - simply writing
```
QStringList types;
```
will initialize to the empty list.

Reply:
Done.